spec/acceptance/foreman_openbolt_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper_acceptance'
+
+describe 'Scenario: install foreman with openbolt' do
+  before(:context) { purge_foreman }
+
+  it_behaves_like 'an idempotent resource' do
+    let(:manifest) do
+      <<-PUPPET
+      include foreman
+      include foreman::plugin::openbolt
+      PUPPET
+    end
+  end
+
+  it_behaves_like 'the foreman application'
+
+  describe curl_command("https://#{host_inventory['fqdn']}/api/plugins", cacert: '/etc/foreman-certs/certificate.pem', user: 'admin:changeme') do
```

Comment:
```suggestion
  describe curl_command("https://admin:changeme@#{host_inventory['fqdn']}/api/plugins", cacert: '/etc/foreman-certs/certificate.pem') do
```

and boom, you don't need `user` ;)